lib/iex/lib/iex/autocomplete.ex

```diff
@@ -225,8 +225,10 @@ defmodule IEx.Autocomplete do
 
     for mod <- match_modules(base, module === Elixir),
         parts = String.split(mod, "."),
-        depth <= length(parts) do
-      %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
+        depth <= length(parts),
+        name = Enum.at(parts, depth - 1),
+        not String.starts_with?(inspect(String.to_atom("Elixir." <> name)), ":") do
```

Comment:
Does not work as expected, many tests fail. Also `Macro.classify_identifier` expects an atom, so we'd still need to call `String.to_atom`.